src/test/lassipatanen/Job/JobPosting.py

```diff
@@ -0,0 +1,136 @@
+from datetime import datetime
+from typing import List, Optional
+
+from definition_tooling.converter import CamelCaseModel, DataProductDefinition
+from pydantic import Field
+
+
+class Location(CamelCaseModel):
+    countries: Optional[List[str]] = Field(
+        None,
+        title="Countries",
+        description="Country codes of the location as the numeric codes according to ISO 3166",
+    )
+    regions: Optional[List[str]] = Field(
+        None,
+        title="Regions",
+        description="The numeric codes according to Statistics Finland's classification of Regions 2022 are used as the code for the region data",
+    )
+    municipalities: Optional[List[str]] = Field(
+        None,
+        title="Municipalities",
+        description="The code of the municipality of location data is the numerical codes of municipalities according to Statistics Finland's municipalities 2022 classification",
+    )
+
+
+class Requirement(CamelCaseModel):
+    occupations: Optional[List[str]] = Field(
+        None,
+        title="Occupations",
+        description="Professions being sought. The occupation codes used are in an URL format according to ESCO 1.1.0",
+    )
+    skills: Optional[List[str]] = Field(
+        None,
+        title="Skills",
+        description="Areas of expertise being sought. The codes for skills are in an URL format according to ESCO 1.1.0",
+    )
+
+
+class Paging(CamelCaseModel):
+    limit: Optional[int] = Field(None, title="Limit", description="Paging limit")
+    offset: Optional[int] = Field(None, title="Offset", description="Paging offset")
+
+
+class JobLocation(CamelCaseModel):
+    municipality: str = Field(
+        ...,
+        title="Municipality",
+        description="The code of the municipality according to Statistics Finland's municipalities 2022 classification",
+    )
+    postcode: Optional[str] = Field(
+        None,
+        title="Postal code",
+        description="Postal code of the location",
+    )
+
+
+class BasicInfo(CamelCaseModel):
+    title: str = Field(
+        ...,
+        title="Title",
+        description="Title of the job posting",
+    )
+    description: str = Field(
+        ...,
+        title="Description",
+        description="Short description about the job",
+    )
+    work_time_type: Optional[str] = Field(
+        None,
+        title="Work time type",
+        description="01 - Full time, 02 - Part time",
```

Comment:
This one is also missing the example and the pre-formatted description has changed.
<img width="852" alt="image" src="https://github.com/Virtual-Finland/definitions/assets/6637576/8f2c51cf-ff84-43aa-afa1-cf38e1f1cc78">

```suggestion
        description="<pre>01 = Full time\n02 = Part time</pre>",
        example="01",
```